src/libra_py/pdos.py

```diff
@@ -384,4 +384,154 @@ def libra_pdos(_emin, _emax, _de, projections, prefix, outfile, Nel, do_convolve
 
     return res
     
-        
\ No newline at end of file
+
+def convolve_cp2k_pdos(cp2k_log_file: str, time_step: int, sigma: float, coef: float, npoints: int, energy_conversion: float, angular_momentum_cols: list):

```

Comment:
there are relatively many input parameters, so could you please pack them into a dictionary. This way you can use the check_input function (please do) to setup the default values and check for critical inputs

also - do you really need the "energy_conversion" as an input parameter? it is merely a constant. for consistency, please use the "libra_py.units" module